tests/test_price_deviation_coingecko.py

```diff
@@ -76,3 +78,9 @@ def test_price_deviation_coingecko():
     check_price_deviation_coingecko(
         100, 110, product_attrs, "ZZZT is more than 5% off from CoinGecko"
     )
+
+
+@pytest.mark.asyncio
+async def test_get_coingecko_prices():
```

Comment:
in general, it's not a good idea to have tests with external API dependencies. The tests become unreliable and hard to run in different settings. I suggest deleting this.

Reply:
deleted